contributing-guides/style-guide.md

```diff
@@ -330,7 +330,16 @@ For instance, instead of `Listing all files:`, `List all files:` can be used as
  `ls`
 ```
 
-## Emphasis
+## Not rendered syntax
+
+Any syntax not documented on this guide or in the client specification will **not** be rendered in our clients. See common cases below.
+
+### Links and images
+
+- Do not use Markdown links (\[link text\](link url)) or images in the pages, they are not supported by our clients.
```

Comment:
```suggestion
Any syntax not documented on this guide or in the client specification will **not** be rendered in most of our clients. See common cases below.

### Links and images

- Do not use Markdown links (\[link text\](link url)) or images in the pages, they are not part of the tldr client specification and thus not supported by most of our clients.
```

I would refrain from making absolute statements, as we ultimately do not have control over the behaviour of our clients (apart from the things required in the client specification).

Also, the specification states that our pages are written in CommonMark (https://commonmark.org/help/), which explicitly includes the mentioned Link syntax. I would be surprised if none of the clients would render such links as described by CommonMark.